pkgs/applications/video/gnome-subtitles/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, fetchFromGitLab
+, autoreconfHook
+, pkg-config
+, intltool
+, itstool
+, mono
+, yelp-tools
+, wrapGAppsHook
+, enchant1
+, glib
+, gnome3
+, gtk-sharp-3_0
+, gst_all_1
+, libxml2
+}:
+let
+  version = "1.6";
+in
+stdenv.mkDerivation {
+  pname = "gnome-subtitles";
+  inherit version;
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-subtitles";
+    rev = "gnome-subtitles-${version}";
+    sha256 = "1crmmcx32i6ca7dlr3xhnc7vgv9jhlpwh6hxhv2fl1x1zbasf42z";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    mono
+    intltool
```

Comment:
Did this still not went through https://wiki.gnome.org/Initiatives/GnomeGoals/GettextMigration?

Reply:
This is an older release. The code in the master branch has been through some major refactoring, so the 1.6 release might not adhere to that page.